app/src/main/res/layout-land/profile_reset_pin_activity.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The best way to create landscape mode is not by writing the entire file from scratch, instead, you should just copy the entire code from portrait mode and paste that in landscape mode file.

Reply:
@rt4914 will keep that in mind, actually, there was an option in the preview pane of layout to create a landscape copy so I used that functionality of android studio.